modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -150,7 +150,7 @@ spec:
         bearertokenauth:
           filename: "/var/run/secrets/kubernetes.io/serviceaccount/token"
       exporters:
-        otlp/dev:
+        otlp/dev: # <1>
```

Comment:
Outside the scope of this work, but the Introduction to TempoStack configuration parameters has an example with, IMO, too many callouts. Maybe a table could be utilized and this would be a good ContentX task.